src/linter/report.go

```diff
@@ -650,6 +650,18 @@ $_ = is_real($a);`,
 			After: `$a = (float)100;
 $_ = is_float($a);`,
 		},
+
+		{
+			Name:     "typeHint",
+			Default:  false,
```

Comment:
It seems to me that this check is too pedantic to be enabled by default.

Reply:
Agreed.